pybandits/quantitative_model/bnn.py

```
@@ -0,0 +1,394 @@
+    """
+
+    bnn: BayesianNeuralNetwork
+
+    @classmethod
+    @validate_call
+    def cold_start(
+        cls,
+        dimension: PositiveInt = 1,
+        n_features: NonNegativeInt = 1,
+        categorical_features: Optional[Dict[NonNegativeInt, NonNegativeInt]] = None,
+        base_model_cold_start_kwargs: Optional[Dict[str, Any]] = Field(default_factory=dict),
```

Ruff (0.15.15)

[warning] Do not perform function call `Field` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)